tpp_database_utils/main.py

```diff
@@ -0,0 +1,43 @@
+import os
+import sys
+
+from custom_medication_dictionary import (
+    CustomMedicationDictionary,
+)
+from maintenance_mode import in_maintenance_mode
+from mssql_connection import pymssql_connect
+
+
+def main():
+    if len(sys.argv) < 2:
+        print("Please specify a command to run")
+        return
+
+    command = sys.argv[1]
+
+    tpp_connection = pymssql_connect()
+    match command:
```

Comment:
This doesn't really matter at this scale, but I think I'd generally prefer to see this as three separate tiny functions and then have the main function doing nothing but argument handling. Obviously I get that this is all intended to be very minimal and not have loads of extra bureaucracy, which I appreciate. But purely as a matter as taste I would probably have added that layer of separation. Happy to let it stay as is though.

Reply:
happy to do this, will update shortly